src/storybook/pages/home/home.common.ts

```diff
@@ -41,20 +35,10 @@ export const timetableInput = (): TemplateResult => html`
   </section>
 `;
 
-const onNavigationClose = (dialog: SbbNavigationElement): void => {
-  dialog?.addEventListener('did-close', () => {
-    (document.getElementById('nav-marker') as SbbNavigationMarkerElement).reset();
-    (document.getElementById('nav-1') as SbbNavigationButtonElement).setAttribute('active', '');
-  });
-};
-
 export const navigation = (): TemplateResult => html`
-  <sbb-navigation
-    trigger="hamburger-menu"
-    ${ref((dialog?: Element) => onNavigationClose(dialog as SbbNavigationElement))}
-  >
+  <sbb-navigation trigger="hamburger-menu">
     <sbb-navigation-marker id="nav-marker">
-      <sbb-navigation-button aria-current="page" id="nav-1" active>
+      <sbb-navigation-button aria-current="page" id="nav-1">
```

Comment:
Shouldn't we use sbb-active here? It somehow doesn't make sense to land on language selection.
```suggestion
      <sbb-navigation-button aria-current="page" id="nav-1"  class="sbb-active">
```